pagopa/pn-frontend
# Fix: handled submit events for courtesy and legal contacts

## Short description
handled submit events for courtesy and legal contacts.

## List of changes proposed in this pull request
- Added onSubmit handlers on FORM html element in courtesy and legal contacts in PF

## How to test
Hitting Enter when you add / edit email, pec or phone will no longer reload the page.
